fetch_request.go

```diff
@@ -32,6 +36,9 @@ func (r *FetchRequest) encode(pe packetEncoder) (err error) {
 	pe.putInt32(-1) // replica ID is always -1 for clients
 	pe.putInt32(r.MaxWaitTime)
 	pe.putInt32(r.MinBytes)
+	if 3 == r.Version {
```

Comment:
`r.Version == 3` is more consistent with the rest of the code, and with the check you added below